interfaces/IWear/include/Wearable/IWear/IWear.h

```diff
@@ -54,15 +52,51 @@ namespace wearable {
     template <typename S>
     using VectorOfSensorPtr = SensorVector<SensorPtr<S>>;
 
-    using WearStatus = sensor::SensorStatus;
     using VectorOfSensorNames = SensorVector<sensor::SensorName>;
 
+    // Wearable device (referes to either sensors or actuator) generic variables
+    template <typename D>
+    using DeviceVector = std::vector<D>;
```

Comment:
Following https://github.com/robotology/wearables/pull/105/files#r558397036, may be `ElementVector` is a cleaner name than `DeviceVector`

Reply:
Done in https://github.com/robotology/wearables/pull/105/commits/ec5c57d1e4aaa41a97e520ceaa10824159b95b80